boards/ark/fmu-v6x/firmware.prototype

```diff
@@ -0,0 +1,13 @@
+{
+    "board_id": 57,
```

Comment:
@AlexKlimaj did you do a reservation PR for this ID in the bootloader repo https://github.com/PX4/PX4-Bootloader/blob/main/board_types.txt#L13?

Reply:
https://github.com/PX4/PX4-Bootloader/pull/220